api/tests/integration/AuthTest.php

```diff
@@ -569,7 +530,7 @@ public function testSingleSignOnVanillaWithUser()
         $token = $this->tokenFromUser($user);
 
         $params = ['client_id' => env('VANILLA_JSCONNECT_CLIENT_ID')];
-        $cookies = [\App\Http\Controllers\AuthController::JWT_AUTH_TOKEN_COOKIE => $token];
+        $cookies = ['ngJwtAuthToken' => $token];
```

Comment:
ok not env, as that is for variables that might change between environments, but a config file sure.

See https://github.com/spira/spira/pull/240/files#diff-c96bc86afb453652f155588526ea4069 for a recent example of custom config file.